src/backend/ai-reviewer-api/src/main/java/ca/bc/gov/open/jag/aireviewerapi/document/DocumentsApiDelegateImpl.java

```diff
@@ -98,7 +100,11 @@ public DocumentsApiDelegateImpl(
         //Due to a bug in mapstruct logic has been performed inline
         ExtractRequest extractRequest = extractRequestMapper.toExtractRequest(extractRequestMapper.toExtract(xTransactionId, (xUseWebhook == null || xUseWebhook)), xDocumentType, file, receivedTimeMillis);
 
+        logger.info("document submission to diligen start, filename: " + file.getName());
+        stopWatch.start();
         BigDecimal response = diligenService.postDocument(xDocumentType, file, documentTypeConfiguration.getProjectId());
+        stopWatch.stop();
+        logger.info("document submission to diligen finish, filename: " + file.getName() + ", size: " + file.getSize() + " bytes, time: " + stopWatch.getLastTaskTimeMillis() + " ms");        
```

Comment:
The logger call can be cleaned up a little further by using args implementation like logger.info("document submission to diligen finish, filename: {}, size: {} bytes, time: {} ms)